lib/api/actions/functions.php

```diff
@@ -489,17 +481,21 @@ function _beans_get_action( $id, $status ) {
 function _beans_set_action( $id, $action, $status, $overwrite = false ) {
 	$id = _beans_unique_action_id( $id );
 
-	// Get the action, if it's already registered.
-	$registered_action = _beans_get_action( $id, $status );
+	// If not overwriting, let's check if an action is already registered for that id.
```

Comment:
`ID` is used three times in the docblock, let's keep it `ID` :-)
`...for that ID.`